management/server/http/policies_handler.go

```diff
@@ -261,12 +312,15 @@ func (h *Policies) GetPolicy(w http.ResponseWriter, r *http.Request) {
 func toPolicyResponse(account *server.Account, policy *server.Policy) *api.Policy {
 	cache := make(map[string]api.GroupMinimum)
 	ap := &api.Policy{
-		Id:          policy.ID,
 		Name:        policy.Name,
 		Description: policy.Description,
 		Enabled:     policy.Enabled,
 		Query:       policy.Query,
 	}
+	if policy.ID != "" {
```

Comment:
policy id should never be empty, right?

Reply:
Fixed

Reply:
Policy ID should never be empty, so we don't need to check it.

Reply:
Fixed